nodejs/eks/nodegroup.ts

```diff
@@ -60,6 +60,10 @@ export interface NodeGroupBaseOptions {
 
     /**
      * The security group to use for all nodes in this worker node group.
+     *
+     * Note: If you want to tag this `nodeSecurityGroup`, you must manually
+     * tag it during its external creation, before it is used in this node
+     * group.
```

Comment:
Perhaps phrase this as `nodeSecurityGroup` and `nodeSecurityGroupTags` are mutually exclusive?  We could also test that at most one is set and error.

Reply:
@lukehoban both the option comment, and the mutual exclusion check have been addressed. PTAL